source/reference/limitations.txt

```diff
@@ -32,16 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same major and minor
-  release version, but can have different patch releases.
-
-  For example,
-  - ``mongosync`` supports sync from a MongoDB 6.0.8 source cluster to a
-    destination cluster using MongoDB 6.0.9, since these are patch releases 
-    of the same major release.
-  - ``mongosync`` does not support sync from a MongoDB 6.0.9 source cluster to 
-    a destination cluster using MongoDB 7.0.0, since they have different major
-    versions.
+- The source and destination clusters must have the same major, minor, and
```

Comment:
Could we revert this to say `The source and destination clusters must have the same release
  version.` ?

Reply:
Fixed